samples/scala-eventsourced-customer-registry/src/main/scala/customer/view/CustomerByNameView.scala

```diff
@@ -18,7 +18,7 @@ class CustomerByNameView(context: ViewContext) extends AbstractCustomerByNameVie
 
   override def processCustomerCreated(
     state: Customer, customerCreated: CustomerCreated): UpdateEffect[Customer] = // <3>
-    if (state == emptyState) effects.ignore() // already  created
+    if (state != emptyState) effects.ignore() // already  created
```

Comment:
may as well remove the extra space in the comment too

```suggestion
    if (state != emptyState) effects.ignore() // already created
```